modules/apm/src/main/java/org/elasticsearch/telemetry/apm/tracing/APMTracer.java

```diff
@@ -98,7 +98,7 @@ public APMTracer(Settings settings) {
         this.enabled = APM_ENABLED_SETTING.get(settings);
     }
 
-    void setEnabled(boolean enabled) {
+    public void setEnabled(boolean enabled) {
```

Comment:
Why change the visibility?

Reply:
I moved APMTracer into its own package (I don't want it to be on an exported package). 
The setEnabled method is called from the APM class (on the exported package)